src/scheduling/dimacs_change_stats.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SRC_SCHEDULING_DIMACS_CHANGE_STATS_H_
+#define SRC_SCHEDULING_DIMACS_CHANGE_STATS_H_
```

Comment:
Remove SRC_. I'll stop commenting about these. We'll just have to do a pass over all the files.